### ISSUE TITLE
Alt text is not descriptive [auth pages]

### ISSUE THREAD

**Issue by timngyn** [aws-amplify/docs#7061]
**Describe the content issue**:
Images listed below do not have descriptive/informative alt text. For markdown the alt text is located between the square brackets, `![alt text](...)`.

The list below isn't exhaustive since I found these images based on the word count for the alt text, but it should be a good starting point to make our alt text better for people who use screen readers.

Some resources to help write alt text:
- https://webaim.org/techniques/alttext/
- https://www.a11yproject.com/posts/are-you-making-these-five-mistakes-when-writing-alt-text/

&nbsp;

https://github.com/aws-amplify/docs/blob/main/src/fragments/lib/auth/js/mfa.mdx:
```
  475  
  476: ![cup](/images/cognito_user_pool_settings.png)
```

&nbsp;

https://github.com/aws-amplify/docs/blob/main/src/fragments/lib/auth/js/overview.mdx:
```
  23  
  24: ![Image](/images/SimpleAuthZ.png)
  25  

  29  
  30: ![Image](/images/SocialAuthZ.png)
  31  

  37  
  38: ![Image](/images/AWSAuthZ.png)
```

&nbsp;

https://github.com/aws-amplify/docs/blob/main/src/fragments/lib/auth/js/react-native-mfa.mdx:
```
  200  
  201: ![cup](/images/cognito_user_pool_settings.png)
```

&nbsp;

https://github.com/aws-amplify/docs/blob/main/src/fragments/lib/auth/native_common/device_features/common.mdx:
```
  26  
  27: ![auth](/images/auth/webconsole_remember1.png)
  28  

  32  
  33: ![auth](/images/auth/webconsole_remember2.png)
  34  
```

&nbsp;

https://github.com/aws-amplify/docs/blob/main/src/fragments/lib-v1/auth/common/device_features/common.mdx:
```
  26  
  27: ![auth](/images/auth/webconsole_remember1.png)
  28  

  32  
  33: ![auth](/images/auth/webconsole_remember2.png)
  34  
```

&nbsp;

https://github.com/aws-amplify/docs/blob/main/src/fragments/lib-v1/auth/common/sms/flows.mdx:
```
  33  
  34: ![Origination Entity warning](/images/sns_origination_entity.png)
  35  
```

&nbsp;

https://github.com/aws-amplify/docs/blob/main/src/fragments/lib-v1/auth/common/social_signin_web_ui/configure_auth_category.mdx:
```
   9  2. From the *My Apps* menu, choose *Your App*.
  10: ![Image](/images/cognitoHostedUI/facebook1.png)
  11  3. On the left navigation bar, choose *Settings* and then *Basic*.
  12: ![Image](/images/cognitoHostedUI/facebook3.png)
  13  4. Choose *+ Add Platform* from the bottom of the page, then choose the relevant platform and fill in the fields.
  14  
  15: ![Image](/images/cognitoHostedUI/facebook9.png)
  16  > Note: For Website, the Site URL should be the location of your [web based production application](https://developers.facebook.com/docs/development/create-an-app/app-dashboard/platform-settings/#website). Facebook will perform a check on this URL and a failure may result in your Facebook app becoming disabled.

  22  
  23:     ![Image](/images/cognitoHostedUI/facebook6.png)
  24  6. Save changes.
  25  8. From the navigation bar choose *Add Product* and then *Set up* from *Facebook Login*.
  26: ![Image](/images/cognitoHostedUI/facebook7.png)
  27  9. From the navigation bar choose *Facebook Login* and then *Settings*.

  31  
  32:     ![Image](/images/cognitoHostedUI/facebook8.png)
  33  11. Save changes.

  43  
  44:     ![Image](/images/cognitoHostedUI/google8.png)
  45  

  53  2. Hover over the gear and choose Web Settings associated with the security profile you created in the previous step, and then choose Edit.
  54: ![Image](/images/cognitoHostedUI/amazon4.png)
  55  3. Type your user pool domain into Allowed Origins and type your user pool domain with the /oauth2/idpresponse endpoint into Allowed Return URLs.
  56: ![Image](/images/cognitoHostedUI/amazon5.png)
  57  5. Choose Save.

  69  8. Type your user pool domain with the `/oauth2/idpresponse` endpoint into **Return URLs**.
  70: ![Image](/images/cognitoHostedUI/apple3.png)
  71  9. Click **Next**, review the information, then select **Done**.

  76  14. Enable **Sign in with Apple** and select **Configure**
  77: ![Image](/images/cognitoHostedUI/apple4.png)
  78  15. Under **Primary App ID** select the app id that was created before.

  81  18. On the page you are redirected to take note of the Key ID and download the .p8 file containing the private key.
  82: ![Image](/images/cognitoHostedUI/apple5.png)
  83  
```

https://github.com/aws-amplify/docs/blob/main/src/fragments/lib-v1/auth/common/social_signin_web_ui/setup_auth_provider.mdx:
```
  10  3. From the *My Apps* menu, choose *Add New App*.
  11: ![Image](/images/cognitoHostedUI/facebook1.png)
  12  
  13  4. Give your Facebook app a name and choose *Create App ID*.
  14: ![Image](/images/cognitoHostedUI/facebook2.png)
  15  
  16  5. On the left navigation bar, choose *Settings* and then *Basic*.
  17: ![Image](/images/cognitoHostedUI/facebook3.png)
  18  

  25  2. Click *Select a project*
  26: ![Image](/images/cognitoHostedUI/google1.png)
  27  3. Click *NEW PROJECT*
  28: ![Image](/images/cognitoHostedUI/google2.png)
  29  4. Type in project name and click *CREATE*
  30: ![Image](/images/cognitoHostedUI/google3.png)
  31  5. Once the project is created, from the left *Navigation menu*, select *APIs & Services*, then select *Credentials*
  32: ![Image](/images/cognitoHostedUI/google4.png)
  33  6. Click *CONFIGURE CONSENT SCREEN*
  34: ![Image](/images/cognitoHostedUI/google5.png)
  35  7. Click *CREATE*
  36: ![Image](/images/cognitoHostedUI/google6.png)
  37  8. Type in *App Information* and *Developer contact information* which are required field and click *SAVE AND CONTINUE* three times (OAuth consent screen -> Scopes -> Test Users) to finish setting up consent screen
  38  9. Back to *Credentials* tab, Create your OAuth2.0 credentials by choosing *OAuth client ID* from the *Create credentials* drop-down list.
  39: ![Image](/images/cognitoHostedUI/google7.png).
  40  10. Choose *Web application* as *Application type* and name your OAuth Client.

  50  3. You need to create an Amazon security profile to receive the Amazon client ID and client secret. Choose Create a Security Profile.
  51: ![Image](/images/cognitoHostedUI/amazon1.png)
  52  4. Type in a Security Profile Name, a Security Profile Description, and a Consent Privacy Notice URL.
  53: ![Image](/images/cognitoHostedUI/amazon2.png)
  54  5. Choose Save.
  55  6. Choose Client ID and Client Secret to show the client ID and secret. You will need them for the next section in the CLI flow.
  56: ![Image](/images/cognitoHostedUI/amazon3.png)
  57  

  68  7. Provide a description in the Description text box and provide the bundleID of the iOS app.
  69: ![Image](/images/cognitoHostedUI/apple1.png)
  70  8. Under Capabilities, select Sign in with Apple.

  74  12. Provide a description in the *Description* text box and provide an identifier for the service id.
  75: ![Image](/images/cognitoHostedUI/apple2.png)
  76  13. Continue and register the service id.
```

&nbsp;

https://github.com/aws-amplify/docs/blob/main/src/fragments/lib-v1/auth/native_common/device_features/common.mdx:
```
  26  
  27: ![auth](/images/auth/webconsole_remember1.png)
  28  

  32  
  33: ![auth](/images/auth/webconsole_remember2.png)
  34  
```

&nbsp;

https://github.com/aws-amplify/docs/blob/main/src/fragments/sdk/auth/ios/federated-identities.mdx:
```
  218  
  219: ![Image](/images/dev-auth-ids-console-settings.png)
  220  
```

&nbsp;

https://github.com/aws-amplify/docs/blob/main/src/pages/[platform]/build-a-backend/auth/add-social-provider/index.mdx:
```
  65  
  66: ⟪ 466 characters skipped ⟫se `signInWithRedirect()` in your app to either show a pre-built "Hosted UI" or pass in a provider name (e.g. `signInWithRedirect({provider:'Facebook'})`) to interface directly and build your own UI. ![Image](/images/SocialAuthZ.png)
  67  
```

&nbsp;

https://github.com/aws-amplify/docs/blob/main/src/pages/[platform]/prev/build-a-backend/auth/add-social-provider/index.mdx:
```
  84  
  85: ⟪ 471 characters skipped ⟫uth.federatedSignIn()` in your app to either show a pre-built "Hosted UI" or pass in a provider name (e.g. `Auth.federatedSignIn({provider: 'Facebook'})`) to interface directly and build your own UI. ![Image](/images/SocialAuthZ.png) Alternatively, you can also get credentials directly from Identity Pools by passing tokens from a social provider directly with `Auth.federatedSignIn()`. However you will have to use that provider's SDK in your app as well as manage the tokens from the provider and their refresh cycles manually.
  86  
```

&nbsp;

https://github.com/aws-amplify/docs/blob/main/src/pages/[platform]/prev/build-a-backend/auth/under-the-hood/index.mdx:
```
  87  
  88: ⟪ 312 characters skipped ⟫nUp` and `Auth.signIn` (or an Amplify UI component) to complete this process and retrieve tokens. The Amplify client will refresh the tokens calling `Auth.currentSession` if they are no longer valid. ![Image](/images/SimpleAuthZ.png)
  89  

  91  
  92: ⟪ 482 characters skipped ⟫dSignIn()` in your application to either show a pre-built "Hosted UI" or pass in a provider name (e.g. `Auth.federatedSignIn({provider: 'Facebook'})`) to interface directly and build out your own UI. ![Image](/images/SocialAuthZ.png) You can also get credentials directly from Identity Pools by passing tokens from a provider directly to `Auth.federatedSignIn()`. However you will have to use that provider's SDK directly in your app and manage token refresh and auth flows manually.
  93  

  95  
  96: ⟪ 577 characters skipped ⟫diagram below shows how JWT tokens are returned from User Pools and AWS credentials from Identity Pools. You can access these at any time with `Auth.currentSession()` and `Auth.currentCredentials()`. ![Image](/images/AWSAuthZ.png)
  97  
```

&nbsp;

https://github.com/aws-amplify/docs/blob/main/src/pages/gen2/build-a-backend/auth/add-social-provider/index.mdx:
```
  25  
  26: ⟪ 430 characters skipped ⟫`signInWithRedirect()` in your app to either show a pre-built "Hosted UI" or pass in a provider name (e.g. `signInWithRedirect({ provider: 'Facebook' })`) to interface directly and build your own UI. ![Image](/images/SocialAuthZ.png)
  27  
```
